.github/workflows/check-milestone.yml

```diff
@@ -19,17 +19,12 @@ jobs:
       - uses: actions/github-script@v7
         with:
           script: | # js
+            const { getLinkedIssues } = require('${{ github.workspace }}/release/utils/index.cjs');
```

Comment:
js/cjs extension mismatch here

Reply:
I think I tried js in my fork and it didn't work out of the box.